zmon_worker_monitor/zmon_worker/notifications/google_hangouts_chat.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import json
+import traceback
+
+import requests
+
+from urllib2 import urlparse
+
+from opentracing_utils import trace, extract_span_from_kwargs
+
+from notification import BaseNotification
+
+logger = logging.getLogger(__name__)
+
+
+class NotifyGoogleHangoutsChat(BaseNotification):
+    @classmethod
+    @trace(operation_name='notification_google_hangouts_chat',
+           pass_span=True,
+           tags={'notification': 'google_hangouts_chat'})
+    def notify(cls, alert, *args, **kwargs):
+
+        current_span = extract_span_from_kwargs(**kwargs)
+
+        webhook_link = kwargs.get('webhook_link', 'http://no.webhook.link?wrong')
```

Comment:
please provide a way to set the parameters and the room id in the config, similar to the hipchat's         
```python
token = kwargs.get('token', cls._config.get('notifications.hipchat.token'))
```

Reply:
google chat gives out just a blank url with query params. for the users this will be extra cumbersome to extract all the three (!!!) necessary items from that url and to put them here so we can put them together. are you sure about that?

another alternative: `webhook_link if webhook_link else concat(base_url + room + token)` (pseudo code)

Reply:
I feel Hanno's comment should have been honored? Looks like we make it quite complicated to users and expose the tokens in the zmon alerts?

@drummerwolli @vetinari

Reply:
@Jan-M there are two approaches (afaik) to post to a Google Chat: 1) webhook and 2) a bot. The second solution would probably be the easier one for users, but a) I think this solution is easy to use as well, b) I had limited time to do this, hence the PR description mentions that this should only kickstart things. Feel free to change/challenge anything, for me just important is to get this going asap.